OpenRA.Mods.Common/Widgets/WorldCommandWidget.cs

```diff
@@ -191,15 +191,24 @@ bool CycleBases()
 				.Select(b => b.Actor)
 				.ToList();
 
+			// If no base is found all existing buildings are selected.
 			if (!bases.Any())
-				return true;
+			{
+				bases = world.ActorsWithTrait<TargetableBuilding>()
+				.Where(a => a.Actor.Owner == world.LocalPlayer)
+				.Select(b => b.Actor)
+				.ToList();
```

Comment:
Please add one more level of indentation on the last 3 lines, and then an empty line after this.